Library/ViewModels/DashboardViewModel.swift

```diff
@@ -103,7 +103,7 @@ public final class DashboardViewModel: DashboardViewModelInputs, DashboardViewMo
   DashboardViewModelType {
 
   public init() {
-    let projects = self.viewWillAppearAnimatedProperty.signal.filter(isFalse).ignoreValues()
+    let projects = self.viewWillAppearAnimatedProperty.signal.ignoreValues()
```

Comment:
Removed check for the view appearance animation fro retrieval of the projects.
This will fix the issue that if push happens before we looked at the dashboard for the first time the dashboard appears animated and projects are not fetched.
This will cause refresh of the projects when we navigate back to dashboard in visual stack (from activities or messages). But it will make it more consistent in the project retrieval logic: view appeared - project refetched.